app/services/gemini_entity_enrichment_service.py

```
@@ -0,0 +1,668 @@
+"""
+Gemini Entity Enrichment Service
+Optimized for FakeGuru with caching, Wikipedia integration, and minimal token usage
+"""
+
+import os
+import json
+import hashlib
+import requests
+import urllib.parse
+from typing import Dict, Any, Optional, List, Tuple
```

Ruff (0.11.9)

`typing.Tuple` imported but unused

Remove unused import: `typing.Tuple`

(F401)